gemrb/GUIScripts/bg1/GUICG4.py

```diff
@@ -167,11 +167,13 @@ def OnLoad():
 
 		Button = AbilityWindow.GetControl(i*2+16)
 		Button.OnPress (LeftPress)
+		Button.SetValue(i)
 		Button.SetVarAssoc("Ability", i )
```

Comment:
Using both SetValue and SetVarAssoc does not make sense and they also modify the same data. If the vars are not needed any more, remove the association calls.